apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -140,6 +147,20 @@ export default class SetupChecklist extends Component {
       });
   }
 
+  /**
+   * Update the firmware on the attached board. Currently, only the CPClassic can be flashed.
+   * @return {Promise}
+   */
+  updateBoardFirmware() {
+    this.setState({[STATUS_BOARD_FIRMWARE]: Status.ATTEMPTING});
+    return window.MakerBridge.flashBoardFirmware({
+      boardName: 'circuit-playground-classic',
+      hexPath:
+        'https://s3.amazonaws.com/downloads.code.org/maker/CircuitPlaygroundFirmata.ino.circuitplay32u4.hex',
+      checksum: '766c4fad9088037ab4839b18292be8b1'
```

Comment:
Not blocking merge of this experiment, but don't we want to point to a latest.json file that includes the checksum and a link to the hex file?

Reply:
Updated